fedimint-core/src/api.rs

```diff
@@ -870,6 +870,26 @@ impl InviteCode {
         ])
     }
 
+    // Constructs an InviteCode which contains as many guardian URLs as needed to
```

Comment:
Docstrings use three slashes, otherwise they don't render in docs.rs. It's also nice to use code formatting for struct names in docs.